product_docs/docs/eprs/7/installing/windows.mdx

```diff
@@ -0,0 +1,87 @@
+---
+title: "Installing Replication Server on Windows"
+navTitle: "On Windows"
+redirects:
+- 01_installing_with_stackbuilder
+- /eprs/latest/03_installation/01_installing_with_stackbuilder/
+---
+
+<div id="installing_with_stackbuilder" class="registered_link"></div>
+
+EDB provides an interactive installer for Windows on the [Downloads page](https://www.enterprisedb.com/software-downloads-postgres#replication-server). 
+
+After downloading the installer, you can either invoke it directly or by using Stack Builder (with PostgreSQL) or StackBuilder Plus (with EDB Postgres Advanced Server). 
```

Comment:
I don't think this is quite right.  As far as I know, StackBuilder and StackBuilder Plus cannot be used to invoke the installer you downloaded via the downloads page.  Rather it is use to download the installer and to optionally (default) install the installation file downloaded by it.

Reply:
@nidhibhammar can you help me with the wording here?

Reply:
I think it is correct now.